web/pdf_sidebar.js

```diff
@@ -104,7 +106,7 @@ var PDFSidebar = (function PDFSidebarClosure() {
     reset: function PDFSidebar_reset() {
       this.isInitialViewSet = false;
 
-      this.close();
+      this._hideUINotification(null);
```

Comment:
*I'm sorry, but I'm slightly confused by https://github.com/mozilla/pdf.js/pull/7959#discussion_r96756794.*

The only place which calls [`PDFSidebar_reset`](https://github.com/mozilla/pdf.js/blob/a917443ae6733e77ec0011097ddfb23fa0e740fc/web/pdf_sidebar.js#L104) is the `close` function in [`app.js`](https://github.com/mozilla/pdf.js/blob/a917443ae6733e77ec0011097ddfb23fa0e740fc/web/app.js#L616). Hence the only effect of removing the `this.close()` call from *here*, i.e. inside of `pdf_sidebar.js`, is that the sidebar will not be forced closed when opening a new document in the default viewer. So this change only applis to an edge-case really.

Please note that the [`PDFOutlineViewer_reset`](https://github.com/mozilla/pdf.js/blob/a917443ae6733e77ec0011097ddfb23fa0e740fc/web/pdf_outline_viewer.js#L62)/[`PDFAttachmentViewer_reset`](https://github.com/mozilla/pdf.js/blob/master/web/pdf_attachment_viewer.js#L57) methods does *not* reset the sidebar itself, but rather only the relevant view.

Hopefully this answers your questions! But please ask again if I misunderstood what you meant.